### PR TITLE
Documentation Update for Contribute section

### PR DESCRIPTION
**Pull request by ric394**
- Working on #7751 
- returned CLI and Source tree section pages to draft to remove them from the site but retain them if necessary.
- Merged the information from many of the pages into the one 'how to contribute' page. This should make directing users to the resource simpler.
- Some information was updated to bring up to speed with current processes and policies regarding versioning.

### CODE REVIEW

File: Docs/content/Contribute/HowToContribute.md [hunk 1]
```diff
@@ -3,22 +3,124 @@ title: "How to contribute"
...
+
+## Using Git to make changes
+
+Instructions on how to use Git to make and submit changes can be found <a href="https://git-scm.com/doc" target="_blank">here</a>
+
+A good practice is to always pull changes from the master APSIM repository before starting any new work. This is important to avoid any problems when it comes time to submit your changes.
+Doing this will ensure you have the most up to date version of the APSIM source files.
+
+## Cloning the repository
+
```
Comment (hol353) [inline Docs/content/Contribute/HowToContribute.md:32]: Move cloning to before 'using git to make changes' section. User needs to clone before they can do anything.

File: Docs/content/Contribute/HowToContribute.md [hunk 2]
```diff
@@ -3,22 +3,124 @@ title: "How to contribute"
...
+The first thing to do is to clone the <a href="https://github.com/APSIMInitiative/ApsimX" target="_blank">APSIMInitiative/ApsimX</a> repository. This can be done by using a command line terminal to navigate to the directory you want to download the source code to and run the command: 
+
+	git clone https://github.com/APSIMInitiative/ApsimX.git
+	
+Alternatively, you can use Visual Studio to clone the repository from the start screen
+
+![Clone repo in visual studio](/images/clone_repo_visual_studio.png)
+	
+It is best practice to fork(copy) the APSIM repository and push changes to this before submitting changes to the master version of APSIM. 
+
```
Comment (hol353) [inline Docs/content/Contribute/HowToContribute.md:42]: Add in a screenshot and text for how to do this in Fork.

File: Docs/content/Contribute/HowToContribute.md [hunk 3]
```diff
@@ -3,22 +3,124 @@ title: "How to contribute"
...
+```
+
+To perform the commit:
+
+```
+git commit -m "Commit message"
+```
+
+For additional options/help, use `git help commit` or `man git`
+
```
Comment (hol353) [inline Docs/content/Contribute/HowToContribute.md:80]: Need to show how to do commits in Fork (not just command line)

File: Docs/content/Contribute/HowToContribute.md [hunk 4]
```diff
@@ -27,7 +129,7 @@ To create a fork:
 
 	![Create pull request](/images/create_pull_request.png)
 	
-	- You can reference issue numbers here. 
+	- You can reference issue numbers here. Each pull request must contain a issue number.
```
Comment (hol353) [inline Docs/content/Contribute/HowToContribute.md:133]: Need to be explicit here and say it must have a 'working on #????' or a 'resolves #????' comment.

**Review by hol353 — COMMENTED** [review]
Looks good. Some suggestions below.

**Comment by ric394** [comment]
Suggestions have been actioned.

**Comment by ric394** [comment]
retest this please jenkins

**Comment by ric394** [comment]
retest this please jenkins

**Comment by hol353** [comment]
@zur003 Any idea why this failed? I saw these errors on your PR this morning.

**Comment by ric394** [comment]
retest this please jenkins

**Pull request merged**